src/node/network/block_import/service.rs

```diff
@@ -253,7 +253,23 @@ where
             if let Some(outcome) = outcome {
                 if let Ok(BlockValidation::ValidBlock { block }) = &outcome.result {
                     this.processed_blocks.insert(block.hash);
-                    // TODO(reth hook): If from proxied validators, target EVN peers with ETH broadcast.
+                    // If from proxied validators, target EVN peers with ETH NewBlockHashes.
+                    if let Some(cfg) = crate::node::network::evn::get_global_evn_config() {
+                        let header_ref = &block.block.0.block.header;
+                        let coinbase = header_ref.beneficiary;
+                        if cfg.proxyed_validators.contains(&coinbase) {
+                            if let Some(net) = crate::shared::get_network_handle() {
+                                let peers = crate::node::network::evn_peers::snapshot();
+                                let hn = BlockHashNumber { hash: block.hash, number: header_ref.number };
+                                let hashes = reth_eth_wire_types::broadcast::NewBlockHashes(vec![hn]);
+                                for (peer_id, info) in peers {
+                                    if info.is_evn {
+                                        net.send_eth_message(peer_id, PeerMessage::NewBlockHashes(hashes.clone()));
```

Comment:
Here need send `NewBlock` msg directly, that avoids fetching block again fron the block hash.

Reply:
fixed